README.md

```diff
@@ -23,11 +23,11 @@ The ably-python client has one dependency,
 
 ### From a git url
 
-    pip install git@bitbucket.org:jjwchoy/ably-python.git
+    pip install -e git+https://github.com/ably/ably-python#egg=AblyPython
```

Comment:
Out of interest, what is `#egg=AblyPython` for?  I'm not a Python developer so am not that familiar with `pip` options.  Also, why the `-e`